packages/react/src/content/hooks/useContentPage.js

```diff
@@ -51,7 +51,7 @@ export default (slug, contentType, strategy = 'default') => {
   }, [action, slug, contentType, strategy]);
 
   useEffect(() => {
-    !contentPage && fetchContent();
+    !contentPage && !isLoading && fetchContent();
```

Comment:
Adding `isLoading` to the dependency injection is a fake solution to avoid re-rendering. The problem is that `fetchContent` is not memoized and not stored inside `useEffect`, causing multiple re-renders. The list should contain: `contentPage`, slug, and `fetchContent`.

Reply:
Thanks for find it. It's done